code/game/area/Space_Station_13_areas.dm

```diff
@@ -1799,3 +1799,47 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	area_flags = UNIQUE_AREA // fuc u
 	outdoors = TRUE
 	ambientsounds = SPACE
+
+// CERESTATION
+
+//HALLWAYS
+/area/hallway/secondary/bridges
+	name = "Hallway"
+	icon = 'modular_splurt/icons/turf/areas.dmi'
+	icon_state = "defaulthallway"
+
+/area/hallway/secondary/bridges/com_engi
+	name = "Command-Engineering Hallway Bridge"
+	icon_state = "com_eng"
+
+/area/hallway/secondary/bridges/serv_sci
+	name = "Service-Science Hallway Bridge"
+	icon_state = "serv_sci"
+
+/area/hallway/secondary/bridges/cargo_ai
+	name = "Cargo-AI Hallway Bridge"
+	icon_state = "cargo_ai"
+
+/area/hallway/secondary/bridges/dock_med
+	name = "Dock-Medical Hallway Bridge"
+	icon_state = "dock_med"
+
+/area/hallway/secondary/bridges/engi_med
+	name = "Engineering-Medical Hallway Bridge"
+	icon_state = "engi_med"
+
+/area/hallway/secondary/bridges/serv_engi
+	name = "Service-Engineering Hallway Bridge"
+	icon_state = "serv_engi"
+
+/area/hallway/secondary/bridges/com_serv
+	name = "Command-Service Hallway Bridge"
+	icon_state = "com_serv"
+
+/area/hallway/secondary/bridges/med_cargo
+	name = "Medical-Cargo Hallway Bridge"
+	icon_state = "med_cargo"
+
+/area/hallway/secondary/bridges/sci_dock
+	name = "Science-Dock Hallway Bridge"
+	icon_state = "sci_dock"
```

Comment:
All of this too

Reply:
I do not know how to modularise

Reply:
We also don't have a modular file for that

Reply:
We do, it's just more modularized in the area/areas folder
also it's literally just making or finding a file that emulates the path in the main code files in modular_splurt and moving all the code there.